torch/onnx/symbolic_opset9.py

```diff
@@ -1621,20 +1621,24 @@ def _pad_packed_sequence(g, data, batch_sizes, batch_first, padding_value, total
 
 def randn(g, *shapes):
     shapes_list = list(shapes)
-    shape = sym_help._maybe_get_const(shapes_list[0], "is")
+    shape = sym_help._get_const(shapes_list[0], "is", "randn")
     return g.op('RandomNormal', shape_i=shape)
 
 
 def rand(g, *shapes):
     shapes_list = list(shapes)
-    shape = sym_help._maybe_get_const(shapes_list[0], "is")
+    shape = sym_help._get_const(shapes_list[0], "is", "rand")
     return g.op('RandomUniform', shape_i=shape)
 
 
 def randn_like(g, self, *others):
     return g.op('RandomNormalLike', self)
 
 
+def rand_like(g, self, *others):
```

Comment:
one of the args of these ops is dtype.
Can you add a Cast when necessary to make sure to match the desired type?
And maybe change one of the tests to set the dtype for a type other than float.

Reply:
Sure. Double was another type supported by torch.rand_like. I've added tests for that.